design-system-docs/src/_pages/accessibility-statement.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Accessibility statement
+layout: basic
+---
+
+This accessibility statement refers only to the Design System elements and components.
+
+## Accessibility and Design System
+
+A core value of Citizens Advice is to serve people whoever they are, regardless of their accessibility needs. An accessible design system is the first step towards creating inclusive customer experiences for all users. For that reason we embarked on a journey to improve our accessibility record across our services and products.
+
+The first step on this journey was to create a design system that will help us offer a consistent and accessible experience to all, including clients, advisers, volunteers and all staff members.
+
+## What standards do we target?
+
+The Citizens Advice Design System has been designed, built and tested to meet the WCAG 2.1 AA standards.
+
+## Where is the design system used?
+
+We have decided to introduce the design system to the Citizens Advice public website and AdviserNet gradually, making sure there are no disruptions to our services. So far the ‘Immigration’ and ‘Family’ sections use the design system. It is also used by our Intranet, smart meter checking tool, and referrals products.
```

Comment:
We might want to change this so we don't have to keep coming back and updating the parts of the public advice site that are migrated to content-platform?
```suggestion
We have decided to introduce the design system to the Citizens Advice public website and AdviserNet gradually, making sure there are no disruptions to our services. So far, some sections use the design system. It is also used by our Intranet, smart meter checking tool, and referrals products.
```